include/deal.II/lac/sparse_matrix.h

```diff
@@ -34,6 +34,15 @@ template <typename number> class Vector;
 template <typename number> class FullMatrix;
 template <typename Matrix> class BlockMatrixBase;
 template <typename number> class SparseILU;
+#ifdef DEAL_II_WITH_MPI
+namespace Utilities
+{
+  namespace MPI
+  {
+    template <typename Number> void sum (const SparseMatrix<Number> &, const MPI_Comm &, SparseMatrix<Number> &);
```

Comment:
You probably need to make sure `MPI_Comm` is also declared here.

Reply:
Sure. But just because we have *configured* for MPI doesn't mean that a *declaration* for `MPI_Comm` is visible at this point. You also have to `#include <mpi.h>` somewhere before this point. It's possible that you get it through one of the other header files `#include`d at the top, but we should just do it explicitly.

Reply:
ah, sure

Reply:
> You also have to #include <mpi.h> somewhere before this point.

done.